parsers/FR_O.py

```diff
@@ -203,3 +213,32 @@ def fetch_production(
             }
         )
     return return_list
+
+
+def fetch_price(
+    zone_key: str,
+    session: Optional[Session] = None,
+    target_datetime: Optional[datetime] = None,
+    logger=getLogger(__name__),
+):
+    production_objects, date_string = fetch_data(
+        zone_key, session, target_datetime, logger
+    )
+
+    return_list: List[Dict[str, Any]] = []
+    for production_object in production_objects:
+        price: Union[float, int] = 0
+        for mode_key in production_object:
+            if mode_key in PRICE_MAPPING:
+                price += production_object[mode_key]
```

Comment:
This code makes the price 0 even if any price data is missing. If the price data is missing it should return `None` and not `0`.